tests/unit/s2n_tls13_handshake_test.c

```diff
@@ -258,16 +258,6 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_stuffer_copy(&client_conn->out, &server_conn->header_in, 5));
         EXPECT_SUCCESS(s2n_stuffer_copy(&client_conn->out, &server_conn->in, s2n_stuffer_data_available(&client_conn->out)));
 
-        /* if aead payload is parsed as plaintext, it would be of length 21 */
```

Comment:
Why are we getting rid of this?

Reply:
We do have other tests for `s2n_record_parse_aead`. What is special in this test to keep it? I don't think its much value here.